yarn.lock

```diff
@@ -1378,13 +1378,6 @@
     "@nodelib/fs.scandir" "2.1.5"
     fastq "^1.6.0"
 
-"@rollup/plugin-json@^6.0.0":
```

Comment:
I was not expecting any yarn.lock changes, but they don't look problematic so I think this is ok!